tests/nnstreamer_datarepo/unittest_datareposrc.cc

```diff
@@ -1030,7 +1029,7 @@ TEST (datareposrc, readInvalidSparseTensors)
   GMainLoop *loop;
   GCallback handler = G_CALLBACK (new_data_cb);
   const gchar *str_pipeline
-      = "datareposrc location=audio1.raw json=sparse.json ! tensor_sink name=tensor_sink0";
+      = "datareposrc location=audio1.raw json=sparse.json ! tensor_sink name=tensor_sink0 sync=true";
```

Comment:
Let's not remove old cases to add ```sync=true``` cases. Add new cases and keep the old cases.

Reply:
Thank you. I will add new TC